helpfunctions/utils.py

```diff
@@ -187,6 +187,39 @@ async def removeRoles(self, userID, roleNames, reason=None):
                 1,
             )
 
+    @staticmethod
+    def roundScientificUnitless(number: str | float | int) -> str:
+        """
+        Rounds input number to a value with 4 digits and uses symbols of the unit prefixes in the metric system to do so.
+
+        Keyword arguments:
+        number -- Is an random number as a string, float or int.
+        """
+        if len(str(number)) > 7:
+            number = str(number)[:-6] + 'M'
+        elif len(str(number)) > 6:
+           number = str(round(float(number)/1000000,1)) + 'M'
+        elif len(str(number)) > 4:
+           number = str(number)[:-3] + 'k' 
+        return number
+
+    @staticmethod
+    def roundScientificTime(hours: float) -> str:

```

Comment:
It is not guarantied that `hours` is a float with exactly one decimal place. Instead of `str(hours)[:-2]` rounding `hours is saver`.